doc/machines/index.rst

```diff
@@ -38,12 +40,20 @@ Parameters:
 
 ``user`` (optional, default=root)
 
+``service_id`` (required)
+
 When the SSH token type is assigned to a client, the user specified in the
 user parameter
 can login with the private key of the SSH token.
 
-In the ``sshd_config`` file you need to configure the ``AuthorizedKeysCommand``.
-Set it to::
+The ``service_id`` identifies the SSH servers or group of SSH servers, where the login is allowed to occur.
+
+To facilitate this, the SSH server fetches the manages SSH keys from the privacyIDEA server on demand.
+There are two scripts, that can be used `privacyidea-authorizedkeys` (a Python script) and `privacyidea-fetchssh`
+(a shell script).
```

Comment:
Yeah, this is a different command from privacyideaadm. This is a shell script, which is actually cooler. But which I currently did not modify, since I am not sure, if such thing should be kept in the privacyideaadm package.
On the other hand the python script would be difficult to install (dependencies). This is why in the first iteration I copied the fetchssh and provide a self consistent shell script with the server.

After all such script needs to be distributed to all SSH servers. Thus having a python script with a lot of dependencies could be cumbersome.

I am doing a rewrite. But I also think or hope that this will change till the release of 3.9

Reply:
Done in 0c7f4f89